helm/cas-airflow/templates/networkSecurityPolicy/intra-release-networkSecurityPolicy.yaml

```
@@ -0,0 +1,15 @@
+kind: NetworkSecurityPolicy
+apiVersion: security.devops.gov.bc.ca/v1alpha1
+metadata:
+  name: custom-airflow-intra-release
+  labels:
+    created-by: {{ .Release.Name }}
```

We should use something like this for all the templates
```suggestion
  labels: {{ include "cas-airflow.labels" . | nindent 4 }}
```